src/components/Card/Card.scss

```
@@ -17,18 +20,24 @@
 }
 
 .Header {
-  padding: spacing() spacing() 0;
+  padding-top: spacing();
+  padding-right: spacing();
+  padding-left: spacing();
+  padding-bottom: $header-spacing;
 
```

Is there a reason this isn't using `padding`?

```suggestion
padding: spacing() spacing() $header-spacing spacing();
```

---

Ryan commented to split out the other larger screen view to make it easier to understand which paddings had changed between the two views. I shouldn't have changed the mobile view one though so I have changed those back.